homeassistant/components/meteoalarm/config_flow.py

```diff
@@ -0,0 +1,98 @@
+"""MeteoAlarm config flow."""
+
+from typing import Any
+
+from meteoalertapi import Meteoalert
+import voluptuous as vol
+
+from homeassistant.components.binary_sensor import (
+    PLATFORM_SCHEMA as BINARY_SENSOR_PLATFORM_SCHEMA,
+)
+from homeassistant.config_entries import ConfigFlow, ConfigFlowResult
+from homeassistant.const import CONF_NAME
+import homeassistant.helpers.config_validation as cv
+
+from .const import CONF_COUNTRY, CONF_LANGUAGE, CONF_PROVINCE, DOMAIN, LOGGER
+
+PLATFORM_SCHEMA = BINARY_SENSOR_PLATFORM_SCHEMA.extend(
+    {
+        vol.Required(CONF_COUNTRY): cv.string,
+        vol.Required(CONF_PROVINCE): cv.string,
+        vol.Optional(CONF_LANGUAGE, default="en"): cv.string,
```

Comment:
Do we know what the list of options is? Can we use the country selector maybe?

Reply:
Yes, that makes sense. Do you know of an example that does this?

Reply:
I _think_ buienradar. Otherwise workday uses this iirc